bridges/NordbayernBridge.php

```diff
@@ -88,6 +97,18 @@ private function getUseFullContent($rawContent) {
 		return $content;
 	}
 
+	private function getTeaser($content) {
+		$teaser = $content->find('p[class=article__teaser]', 0);
+		if($teaser == null) {
```

Comment:
```suggestion
		if(is_null($teaser)) {
```

Reply:
I prefer `$foo === null`. It's faster to type. And new php coders don't have to lookup the function to see whether it actually does a comparison with `null` or not. It's also unclear whether the function does a loose or a strict comparison.

Reply:
I don't have a strong opinion on this.